src/Spring.js

```diff
@@ -153,7 +157,7 @@ function mergeDiff(collA, collB, onRemove, accum) {
 }
 
 function mergeDiffObj(a, b, onRemove) {
-  let keys = mergeDiff(Object.keys(a), Object.keys(b), a => !onRemove(a), []);
+  let keys = mergeDiff(Object.keys(a), Object.keys(b), _a => !onRemove(_a), []);
```

Comment:
overrides function argument. here and after - I just added `_`, but ideally they should have some other name that will not clash.